pkg/reconciler/ingress/ingress.go

```diff
@@ -31,28 +31,34 @@ type Reconciler struct {
 }
 
 const (
-	asyncSuffix                     = "-async"
-	newSuffix                       = "-new"
-	preferHeaderField               = "Prefer"
-	preferAsyncValue                = "respond-async"
-	preferSyncValue                 = "respond-sync"
-	asyncFrequencyTypeAnnotationKey = "async.knative.dev/frequency.type"
-	asyncFrequencyType              = "always.async.knative.dev"
-	publicLBDomain                  = "istio-ingressgateway.istio-system.svc.cluster.local"
-	privateLBDomain                 = "cluster-local-gateway.istio-system.svc.cluster.local"
-	producerServiceName             = "producer-service"
+	asyncSuffix            = "-async"
+	newSuffix              = "-new"
+	preferHeaderField      = "Prefer"
+	preferAsyncValue       = "respond-async"
+	preferSyncValue        = "respond-sync"
+	asyncModeAnnotationKey = "async.knative.dev/mode"
```

Comment:
since this is a key and does not change (presumably) then it could be public?

Reply:
And a constant...

Reply:
already a constant, but updated key to be public.